src/main/java/com/liferay/ide/idea/util/LiferayWorkspaceSupport.java

```diff
@@ -329,7 +333,38 @@ public default VirtualFile getModuleExtDirFile(Project project) {
 
 		List<String> javaHomePaths = JavaHomeFinder.suggestHomePaths();
 
+		File javaHomeFile = null;
+
 		if (javaHomePaths.isEmpty()) {
+			String pathEnv = System.getenv("PATH");
+
+			String[] paths = pathEnv.split(Pattern.quote(File.pathSeparator));
+
+			for (String pathValue : paths) {
+				Path path = Paths.get(pathValue);
+
+				Path javaPath = path.resolve("java");
+
+				if (Files.exists(javaPath)) {
+					javaHomeFile = javaPath.toFile();
+
+					javaHomeFile = javaHomeFile.getParentFile();
```

Comment:
Does this work for both JRE and JDKs? since they may have different directory levels?